spec/requests/api/blueprints_spec.rb

```diff
@@ -254,6 +254,26 @@
 
       expect(response).to have_http_status(:forbidden)
     end
+
+    it "can publish multiple blueprints" do
+      blueprint1, blueprint2 = FactoryGirl.create_list(:blueprint, 2)
+      service_template = FactoryGirl.create(:service_template)
+      service_catalog = FactoryGirl.create(:service_template_catalog)
+      service_dialog = FactoryGirl.create(:dialog_with_tab_and_group_and_field)
+      blueprint1.create_bundle(:service_templates => [service_template],
+                               :service_dialog    => service_dialog,
+                               :service_catalog   => service_catalog)
+      blueprint2.create_bundle(:service_templates => [service_template],
+                               :service_dialog    => service_dialog,
+                               :service_catalog   => service_catalog)
+
+      api_basic_authorize collection_action_identifier(:blueprints, :publish)
+
+      run_post(blueprints_url, :action => "publish", :resources => [{:id => blueprint1.id}, {:id => blueprint2.id}])
+
+      expect(blueprint1.reload.published?).to be_truthy
```

Comment:
Extremely minor, but you can leverage some RSpec magic to make this a tiny bit more readable:

```ruby
expect(blueprint1.reload).to be_published
```

Having said that, it may be better to also verify that it was the request that changed the state:

```ruby
expect do
  run_post(:etc => "etc")
end.to change { blueprint1.reload.published }.from(false).to(true).and(change { blueprint2.reload.published }.from(false).to(true))
```

Can you also add tests for the HTTP status and response body? Thanks!